Makefile.am

```diff
@@ -817,6 +817,7 @@ check_PROGRAMS += \
 	tests/test-delete-link \
 	tests/test-socket-creation \
 	tests/test-u32-filter-with-actions \
+	tests/test-ematch-tree-clone \
```

Comment:
please sort the new name alphabetically (yes, it's not entirely sorted yet, but new entries should be sorted. Optionally, a follow-up patch to sort it all would be nice).

Reply:
Done